dbt-snowflake/tests/functional/iceberg/models.py

```diff
@@ -70,8 +70,9 @@
 _MODEL_BASIC_DYNAMIC_TABLE_MODEL_WITH_PATH = """
 {{
   config(
-    transient = "transient",
     materialized = "dynamic_table",
+    snowflake_warehouse='DBT_TESTING',
```

Comment:
It looks like the base was copied from the table materialization model and updated with `dynamic_table` instead of copying from the Iceberg materialization and adding the new `base_location_root` config option.